cpp/src/rolling/rolling.cu

```diff
@@ -629,6 +632,24 @@ std::unique_ptr<column> rolling_window(column_view const& input,
   }
 }
 
+struct window_wrapper {
+
+  const cudf::size_type *d_group_offsets;
+  const cudf::size_type *d_group_labels;
+  cudf::size_type window;
+
+  window_wrapper(
```

Comment:
Any reason not to use list initialization instead of an explicit constructor?

Reply:
You mean using `std::initializer_list`?

Reply:
I do. We use list initialization pretty much anywhere we can in the rest of cudf, but I guess that's preference.

Reply:
I would love to use `std::initializer_list` but this is a code string for `jifity` so could not use that ...